frontend/src/components/RegionPermissionModal.css

```diff
@@ -0,0 +1,21 @@
+.smart-hub--region-permission-modal .usa-modal--lg .usa-modal__main {
+    max-width: 100%;
+    padding: 1.25rem 14% 4rem;
+}
+
+
+.smart-hub--region-permission-modal .usa-modal--lg .usa-modal__heading {
+    font-size: 1.85rem;
+}
+
+.smart-hub--region-permission-modal .popup-modal button {
+    padding: 9px 16px;
```

Comment:
Again, nitpick: we're using the 8px scale most everywhere else, we've got a 9px here, also the font-size: 1.85rem, did that come from somewhere? Would rather use one of the pixel sizes from here if possible: https://designsystem.digital.gov/utilities/font-size-and-family/